src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -213,6 +213,27 @@ function bindEventsAfterAjax() {
     setupFsCopyModal();
 }
 
+function escapeXml(unsafe) {
+    return (unsafe || '').replace(/&/g, '&amp;').replace(/</g, '&lt;')
+            .replace(/>/g, '&gt;').replace(/"/g, '&quot;')
+            .replace(/'/g, '&#039;');
+}
+
+function initializeCourseName() {
+    $('.course-name-data').each((idx, obj) => {
+        $('.course-name-data').data(obj.id, obj.value);
+    });
+    const selectedId = $('#courseid').val();
+    $('#coursename').html(escapeXml($('.course-name-data').data(selectedId)));
```

Comment:
I just checked again, do you really need to `escapeXml` here? The ID and value of `course-name-data` are already sanitized with `fn:escapeXml`.

Reply:
Yes, the value is escaped once before they are stored in the input hidden field, but when I get the value by using JavaScript, it will be the unescaped character, so I will have to sanitize again. 

For example, course name ```A<script>alert("Hi");</script>``` will be stored as ```A<script>alert(&quot;Hi&quot;);</script>```, but when I get the value by JavaScript, it will be ```A<script>alert("Hi");</script>```

This is what I observed, it will be good if anyone  experienced in JavaScript can provide some input on whether this is a expected behaviour of JavaScript.